src/backend/distributed/metadata/metadata_sync.c

```diff
@@ -786,6 +795,16 @@ DistributedObjectMetadataSyncCommandList(void)
 		{
 			colocationIdList = lappend_int(colocationIdList, colocationId);
 		}
+
+		if (forcePushdownIsNull)
+		{
+			forcePushdown = false;
+			forcePushdownList = lappend(forcePushdownList, (bool *) &forcePushdown);
```

Comment:
this seems buggy, maybe just use lappend_int

Reply:
Initially I tried int, but the bool true and false are not 1 and 0 integers but ascii-values 1 and 0. This whole thing looks awkward, let me see how I can rearrange it

Reply:
Simplified it now.